extensions/ql-vscode/src/databases/db-module.ts

```diff
@@ -21,9 +22,9 @@ export class DbModule extends DisposableObject {
 
     void logger.log('Initializing database module');
 
-    const storagePath = extensionContext.storageUri?.fsPath || extensionContext.globalStorageUri.fsPath;
-    const extensionPath = extensionContext.extensionPath;
-    const dbConfigStore = new DbConfigStore(storagePath, extensionPath);
+    const app = ExtensionApp.CreateFromExtensionContext(extensionContext);
```

Comment:
Would it make sense to create this in the `extension.ts` file instead and re-use the same instance for all classes that will eventually use this? That might also make it easier to test this class without having an extension context.